src/driver.cpp

```diff
@@ -12,6 +12,31 @@ extern bpftrace::location loc;
 
 namespace bpftrace {
 
+namespace {
+std::string_view parse_license()
+{
+  std::string_view program = Log::get().get_source();
+  constexpr std::string_view spdx_id_marker = "SPDX-License-Identifier: ";
+  auto spdx_id_pos = program.find(spdx_id_marker);
+  if (spdx_id_pos == program.npos)
+    return {};
+
+  auto license_pos = spdx_id_pos + spdx_id_marker.size();
+  auto eol_pos = program.find("\n", spdx_id_pos + 1);
+  if (eol_pos == program.npos)
+    return {};
+
+  auto license = program.substr(license_pos, eol_pos - license_pos);
+
+  // Try to translate some known GPL-v2-compatible licenses into a format
+  // understood by the Linux kernel
+  if (license.find("GPL-2.0") == 0)
```

Comment:
Should we do something similar for GPL-1.0?

Reply:
Ah this all stems from the fact that I thought the "GPL" string meant GPL v1 but apparently it doesn't 🙄

Reply:
It's definitely confusing!